test/it/ServicesISpec.scala

```diff
@@ -178,10 +176,10 @@ class ServicesISpec extends AnyWordSpec with Matchers with GuiceOneAppPerSuite w
       )
 
       forAll(failingPropertyValues) { (propertyName, propertyValue, errorMsg) =>
-        withYAMLFieldProperty(propertyName, propertyValue).failure.exception shouldBe DecodingFailure(
-          errorMsg,
-          List(DownField(propertyName))
-        )
+        withYAMLFieldProperty(
+          propertyName,
+          propertyValue
+        ).failure.exception.getMessage shouldBe "YamlParser: Failed to decode json result."
```

Comment:
By doing this, do we not swallow the exception message of which fields could not be decoded?